src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -1274,6 +1338,27 @@ private:
     lines.add(kj::str(""));
   }
 
+  static bool checkIsHttpToken(kj::StringPtr val) {
+    // Allowable values for HTTP tokens per RFC 7230 ("token").
+    // Same as valid header field names ("field-name").
+    // Notably, this excludes CR and LF, thus preventing header injection.
+    static const std::regex tokenRe = std::regex("^[a-zA-Z0-9_!#$%&'*+.^`|~-]+$");
```

Comment:
Can you add:

    // TODO(cleanup): Use kj::parse rather than std::regex.

(I won't make you cater to my NIH in this PR, though. :) )